libqfieldsync/offline_converter.py

```diff
@@ -354,11 +362,35 @@ def _convert(self, project: QgsProject) -> None:  # noqa: PLR0912, PLR0915
             project_path = Path(project.fileName()).parent
             for additional_project_file in additional_project_files:
                 additional_project_file_path = Path(additional_project_file)
-                relative_path = additional_project_file_path.relative_to(project_path)
+                additional_project_file_name = additional_project_file_path.name
+
+                additional_project_file_relative_path = (
+                    additional_project_file_path.relative_to(project_path)
+                )
+
                 destination_file = self._export_filename.parent.joinpath(
-                    relative_path
+                    additional_project_file_relative_path
                 ).resolve()
                 destination_file.parent.mkdir(parents=True, exist_ok=True)
+
+                if (
```

Comment:
```suggestion
                # some special type of file we need to manage
                if (
```

Can you add a bit of docs what are these types of files we need special care of?